simphony_ui/ui.py

```diff
@@ -290,5 +272,6 @@ def _openfoam_settings_default(self):
 
 
 if __name__ == '__main__':
```

Comment:
Technically, this part can be removed. You use the other main() function to show the application